runbot_bitbucket/models/bitbucket.py

```diff
@@ -1,16 +1,18 @@
 from openerp.addons.runbot_multiple_hosting.models import hosting
+import requests
+
 
 class BitBucketHosting(hosting.Hosting):
     API_URL = 'https://bitbucket.org/api/2.0'
     URL = 'https://bitbucket.org'
+    token = (())
 
     def __init__(self, credentials):
-        super(BitBucketHosting, self).__init__(credentials)
-
-    def get_pull_request(self, owner, repository, pull_number):
-        url = self.get_api_url('/repositories/%s/%s/pullrequests/%s' % (owner, repository, pull_number))
-        reponse = self.session.get(url)
-        return reponse.json()
+        auth = ('JAFMQPQs4BkT7KQubr','JE54xLmGDFLGqvjKbvKn3NBGGaVAPEC7')
```

Comment:
Why are you making this public? OAuth tokens shouldn't be in the code.

https://confluence.atlassian.com/display/BITBUCKET/OAuth+FAQ#OAuthFAQ-ShouldmysampleOAuthsourcecodeincludemykeys?